src/pages/assets.tsx

```diff
@@ -93,245 +32,253 @@ const AssetsPage = ({ data }: PageProps<Queries.AssetsPageQuery, Context>) => {
     ? data.ethDiamondPurpleHero
     : data.ethDiamondBlackHero
   return (
-    <Page>
+    <Flex direction={"column"} width="full">
       <PageMetadata
         title={translateMessageId("page-assets-meta-title", intl)}
         description={translateMessageId("page-assets-meta-desc", intl)}
       />
-      <Content>
-        <HeroContainer>
-          <Header>
-            <Image
+      <Box padding={"1rem 2rem"}>
```

Comment:
To follow conventions, lets use as much as we can, chakra values. You can see them [here](https://chakra-ui.com/docs/styled-system/theme#spacing).
```suggestion
      <Box py={4} px={8}>
```